broadway_api/daos/worker_node.py

```diff
@@ -6,6 +6,7 @@
 
 class WorkerNodeDao(BaseDao):
     ID = "_id"
+    WORKER_ID = "worker_id"
```

Comment:
Any specific reason you have decided to add a worker ID and keep the `_id` field. Worker ID by definition is supposed to be unique so with this design we have 2 attributes of the object which are unique. Mongo lets us overwrite (or specify) the `_id` field during creation for this reason.

Reply:
ummm I just found that the grading_job dao is doing the same thing and thought there might be some reasons you did that..

Reply:
nvm my bad. I confused run_id with job_id..

Reply:
yeah, every object in the database model currently uses `_id` for identification.